Emby.Naming/Video/VideoListResolver.cs

```diff
@@ -213,9 +209,9 @@ private bool IsEligibleForMultiVersion(string folderName, string testFilename)
             if (testFilename.StartsWith(folderName, StringComparison.OrdinalIgnoreCase))
             {
                 testFilename = testFilename.Substring(folderName.Length).Trim();
-                return string.IsNullOrEmpty(testFilename) ||
-                       testFilename.StartsWith("-") ||
-                       string.IsNullOrWhiteSpace(Regex.Replace(testFilename, @"\[([^]]*)\]", string.Empty)) ;
+                return string.IsNullOrEmpty(testFilename)
+                    || testFilename[0] == '-'
```

Comment:
That's a change in behavior. I'm really unfamiliar with what `IsEligibleForMultiVersion` is. Just wanted to double check that this is the expected behavior.